### PR TITLE
Add Starling Staking delegation token asset metadata and image

### PR DESCRIPTION
**Pull request by plaidfinch**
This PR adds the [Starling Staking](https://starlingcyber.net) delegation token and its asset metadata, along with an SVG suitable for use in token display.

### CODE REVIEW

File: input/penumbra-testnet-deimos-8.json [hunk 1]
```diff
@@ -212,6 +212,32 @@
...
+          "denom": "mdelegation_penumbravalid1gjdvn0u85rgldqk5adfexn6n4y8d2m3tfla54sc4gu95xwpzssxsjutk7u",
+          "exponent": 3
+        },
+        {
+          "denom": "udelegation_penumbravalid1gjdvn0u85rgldqk5adfexn6n4y8d2m3tfla54sc4gu95xwpzssxsjutk7u"
+        }
+      ],
+      "base": "udelegation_penumbravalid1gjdvn0u85rgldqk5adfexn6n4y8d2m3tfla54sc4gu95xwpzssxsjutk7u",
+      "display": "delegation_penumbravalid1gjdvn0u85rgldqk5adfexn6n4y8d2m3tfla54sc4gu95xwpzssxsjutk7u",
+      "symbol": "Delegation (Starling Staking)",
```
Comment (hdevalence) [inline input/penumbra-testnet-deimos-8.json:232]: This should be consistent with the other delegation tokens
```suggestion
      "symbol": "delUM(Starling Staking)",
```

File: input/penumbra-testnet-deimos-8.json [hunk 2]
```diff
@@ -212,6 +212,32 @@
...
+      "penumbraAssetId": {
+        "inner": "/5AHh95RAybBbUhQ5zXMWCvstH4rRK/5KMVIVGQltAw="
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/prax-wallet/registry/main/images/starling-staking.svg"
```
Comment (hdevalence) [inline input/penumbra-testnet-deimos-8.json:238]: This is an SVG, not a PNG.
```suggestion
          "svg": "https://raw.githubusercontent.com/prax-wallet/registry/main/images/starling-staking.svg"
```

**Review by hdevalence — CHANGES_REQUESTED** [review]
The symbol for the delegation token should match the auto-generated symbols used in by Prax.

**Review by hdevalence — APPROVED** [review]

**Comment by grod220** [comment]
The changeset is not done quite right. Will merge this and open another PR that fixes this and cuts a release 👍 

https://github.com/prax-wallet/registry/blob/main/README.md?plain=1#L44-L46

**Comment by grod220** [comment]
Hmm. It kind of feels like it is needed to be done manually given it requires the user to use the changesets CLI to:
- Declare a change that requires a new publish
- Give a summary of that change
- Identify if it was a patch/minor/major

In theory, a github action command could create one manually and use the commit text as the changeset summary and then make another commit with the version bump. Not sure how it would distinguish the level of version bump it needs though 🤔 . Is this the workflow you had in mind though @plaidfinch?

**Pull request merged**